app/controllers/hmis_api/user_controller.rb

```diff
@@ -0,0 +1,31 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class HmisApi::UserController < HmisApi::BaseController
+  skip_before_action :verify_authenticity_token
```

Comment:
Should this only happen on some routes?  Or do we need to skip it entirely?

Reply:
It needs to be skipped only for `GET /hmis-api/user`, because that's the initial request we make on page load to get the csrf token. That's what this is doing, because there's only one action in this controller, but I can make it clearer by adding `, only: [:index]`